docs/wmco-prerequisites.md

```diff
@@ -2,8 +2,15 @@
 Below are the Cluster and OS prerequisites for WMCO. Please also see the vSphere-specific section that 
 may be relevant.
 
-## Supported Cloud Providers based on OKD/OCP Version and WMCO version
-| Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
+## Supported Installation method
+* Installer-Provisioned Infrastructure (IPI) installation method is the recommended and supported installation method.
+
+* User-Provisioned Infrastructure (UPI) is **only supported** on Bare metal or Provider-agnostic
+  platform (`platform: none`) for the [BYOH (Bring Your Own Host)](byoh-instance-pre-requisites.md#byoh-instance-pre-requisites)
+  use case. UPI is not supported in any other platform.
```

Comment:
@jrvaldes Is this too much? (And accurate?)
```suggestion
##Supported OKD/OCP Installation method
Creating Windows nodes is supported for OKD/OCP clusters installed by using the Installer-Provisioned Infrastructure (IPI) and User-Provisioned Infrastructure (UPI) installation methods as follows:

* IPI. Creating Windows nodes on OKD/OCP clusters that used the IPI installation method is supported for all platforms. This is the recommended and OKD/OCP installation method.
* UPI. Creating Windows nodes on OKD/OCP clusters that used the UPI installation method is **only supported** for the [BYOH (Bring Your Own Host)](byoh-instance-pre-requisites.md#byoh-instance-pre-requisites) use case with the bare metal or provider-agnostic platform (`platform: none`). UPI is not supported on any other platform.
```

Reply:
> supported for OKD/OCP clusters that were installed by using the Installer-Provisioned Infrastructure (IPI) and User-Provisioned Infrastructure (UPI) installation methods

@mburke5678   I have mixed feelings about the first statement; it may lead to misunderstands. e.g. `supported for IPI and UPI`

I would rather be concrete and specific.

Reply:
It confused me that we said IPI is the supported installation method, then say that UPI is also supported, although only in specific conditions. 
However, I can see your concern, and perhaps users won't see those specific conditions and  assume UPI is supported in all platforms. 
I'll defer to what the team thinks.

Reply:
@mburke5678 good catch. I reframed it as "is supported and the recommended installation method" so it is consistent. 

I pushed an update. PTAL